checkov/common/runners/runner_registry.py

```diff
@@ -134,7 +134,10 @@ def _parallel_run(runner: _BaseRunner) -> Report | list[Report]:
                     logging.error(f"Failed to create report for {runner.check_type} framework")
                     report = Report(check_type=runner.check_type)
 
-                return report
+                if runner.graph_manager:
+                    check_type_to_graph = {runner.check_type: runner.graph_manager.get_reader_endpoint()}
+                    return report, check_type_to_graph
+                return report, {}
```

Comment:
```suggestion
                if runner.graph_manager:
                    check_type_to_graph = {runner.check_type: runner.graph_manager.get_reader_endpoint()}
                    return report, runner.check_type, runner.graph_manager.get_reader_endpoint()
                return report, None, None
```
how about returning all as a normal tuple, then you can check if both are not `None` and update `full_check_type_to_graph[check_type] = reader_endpoint`

Reply:
fixed